file_paths.md

```diff
@@ -280,6 +280,13 @@ These sections describe other Puppet packages that rely on puppet-agent to creat
                 server_datadir            # :server_data
                 yaml                      # :yamldir
 
+    /var/log/puppetlabs                   # :logdir                      /var/lib/puppet/log
+        puppetserver/                     # writeable by puppetserver
+            puppetserver.log
```

Comment:
So we're saying each server-side app that runs as `puppet` will need to create it's own `/var/log/puppetlabs/$project` directory? Or the `puppetserver` package is going to create a `/var/log/puppetlabs/server` directory owned by `puppet:puppet`, and other server-side apps write to?

I assume the former, because not every server will have a `puppetserver` package installed, e.g. puppetdb, console?